plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy_test.go

```diff
@@ -31,12 +31,17 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/diff"
 	"k8s.io/apimachinery/pkg/util/sets"
+	genericfeatures "k8s.io/apiserver/pkg/features"
```

Comment:
```suggestion
	"k8s.io/apiserver/pkg/features"
```

Reply:
there is already "k8s.io/kubernetes/pkg/features" imported, so I need to avoid a collision. Chosen the `genericfeatures` name as in `controller_policy.go`.

Reply:
removed after refactoring the test to only focus on the PodDisruptionConditions feature gate.